### PR TITLE
tuist 0.2.1

### PR DESCRIPTION
**Pull request by JanC**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Review by danyf90 — APPROVED** [review]

File: steps/tuist/0.2.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
```
Comment (koral--) [inline steps/tuist/0.2.0/step.yml:30]: You are using curl in the step.sh, not git and wget.

Comment (JanC) [reply steps/tuist/0.2.0/step.yml:30]: @danyf90 I was also surprised to see those `wget` and `git` dependencies in the tuist repo yaml while the previous version of the step had `curl` as dependecy.

https://github.com/tuist/bitrise-step-tuist/blob/70c07d42370ea9ec6c19a0c1aa5a5f3cb9c1c4a5/step.yml#L36-L37

I believe those should be changed in the tuist repo as well?

Comment (danyf90) [reply steps/tuist/0.2.0/step.yml:30]: Merged and released 0.2.1

Comment (JanC) [reply steps/tuist/0.2.0/step.yml]: @koral--  Now the step.yml has the correct dependencies

**Review by danyf90 — APPROVED** [review]

**Comment by JanC** [comment]
@koral--  would you mind approving / merging please?

File: steps/tuist/0.2.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+title: tuist
```
Comment (godrei) [inline steps/tuist/0.2.1/step.yml:1]: NOTE: `Run Tuist` was better in line with the Step title requirements: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#the-title

Please consider updating the title in the next release.

**Review by godrei — APPROVED** [review]

**Pull request merged**